src/include/daos_srv/iv.h

```diff
@@ -23,14 +23,20 @@ struct ds_iv_ns {
 	unsigned int	iv_ns_id;
 	/* Link to global ns list (ds_iv_list) */
 	d_list_t	iv_ns_link;
-	/* Protect to the key list */
-	ABT_mutex	iv_lock;
 	/* all of entries under the ns links here */
 	d_list_t	iv_entry_list;
 	/* Cart IV namespace */
 	crt_iv_namespace_t	iv_ns;
 	/* pool uuid */
 	uuid_t		iv_pool_uuid;
+
+	int		iv_refcount;
+	ABT_eventual	iv_done_eventual;
```

Comment:
minor...there are some wasted bytes in the struct because eventual is 64 bits...if you swap line 33 and 34, the issue goes away

Reply:
ah, indeed. will fix it in the following patch. Thanks.